Dockerfile.tnf.ocp

```diff
@@ -0,0 +1,10 @@
+FROM registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.23-openshift-4.19 AS builder
```

Comment:
I wonder if you guys can just grab an entire folder with a dedicated OWNERS file.

Remind me again: if you need a new container image entirely, how is that different to keep your entire operator in a separate repository? or is this just a copy+paste leftover from your PoC?

Reply:
Thank you!